ash/src/extensions/ext/mesh_shader.rs

```diff
@@ -0,0 +1,87 @@
+use crate::vk;
+use crate::{Device, Instance};
+use std::ffi::CStr;
+use std::mem;
+
+#[derive(Clone)]
+pub struct MeshShader {
```

Comment:
Since recently we also link this struct to the extension docs:

```suggestion
/// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_mesh_shader.html>
#[derive(Clone)]
pub struct MeshShader {
```